packages/core/src/conflict-resolution.ts

```diff
@@ -253,10 +273,10 @@ export class ConflictResolution {
         // it's an anchor commit
         // TODO: Anchor validation should be done by the StreamHandler as part of applying the anchor commit
         await verifyAnchorCommit(this.dispatcher, this.anchorService, commit);
-        state = await handler.applyCommit(commit, cid, this.context, state);
+        state = await handler.applyCommit(commit, { cid: cid }, this.context, state);
```

Comment:
Can't we just pass `entry.value[1]` here?
```suggestion
        state = await handler.applyCommit(commit, entry.value[1], this.context, state);
```

Reply:
Maybe we just create a variable `const commitMeta = entry.value[1]`?

Reply:
done.